src/main/java/com/endava/soa_5/exception_and_io/Main.java

```diff
@@ -0,0 +1,66 @@
+package com.endava.soa_5.exception_and_io;
+
+import com.endava.soa_5.array_and_string.Abecedarian;
+import com.endava.soa_5.array_and_string.MaxIndexOfArray;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        SetProperty properties = new SetProperty("Resource.properties");
+        String arrayInString = null, word, word1, word3;
+
+        // This part of code will get value from properties file and will return boolean value if String is abecedarian or not.
+
+        try {
+            try {
```

Comment:
Please create a method for each try catch. When possible, make a common method for more try catch blocks and reuse it.

Reply:
Done! was refactored.